engines/parallaction/parallaction.h

```diff
@@ -219,6 +226,88 @@ struct Character {
 	bool dummy() const;
 };
 
+struct CharacterVoiceData {
+	Common::String characterName;
```

Comment:
maybe use a `const char *` here to reduce the size of the struct as the Common::String has some internal storage that might add empty bytes and wastes memory.

also the `voiceID` could be `uint8`